tests/integration/test_dictionaries_postgresql/configs/users.xml

```diff
@@ -0,0 +1,10 @@
+<clickhouse>
+    <users>
+        <default>
+            <password></password>
+            <profile>default</profile>
+            <quota>default</quota>
+            <use_named_collections>1</use_named_collections>
```

Comment:
Hm, I don't see such directive in the code,  though not sure how it works then...

Reply:
I did not yet apply new code of named collections to all dictionary sources, for postgresql including. But this new access type works only in new code, so yes it does not work for this test. And yes, "use_named_collections" cannot be written in config as it is done here, it is probably a leftover of some intermediate changes.